scripts/run_gcosmos.sh

```diff
@@ -5,13 +5,77 @@ G_HTTP_ADDR=${G_HTTP_ADDR:-"26657"}
 G_GRPC_ADDR=${G_GRPC_ADDR:-"9092"}
 
 # cleanup previous run data as gordian can only start from height 0 currently
-rm -rf ~/.simappv2/
+rm -rf ./test
+mkdir -p ./test/genesis/config/gentx
 
-./gcosmos init moniker --chain-id=${CHAIN_ID}
+trap 'rm -rf ./test; echo; echo "Gordian Cosmos testnet stopped"; exit' INT
```

Comment:
This wouldn't stop the backgrounded other validators, would it?

Reply:
I originally had the kill PID commands here, but it was not necessary. They are killed with Ctrl+C for me on fedora. I'll try on macOS to make sure the behavior is the same.